docker/dev/docker-compose.yml

```
@@ -1,27 +1,47 @@
 version: '3.8'
 
 services:
-  php8.1:
-    image: php:8.1.25RC1-zts-alpine3.18
-  postgresql:
-    image: postgres:16-alpine3.17
-    environment:
-      - POSTGRES_PASSWORD=postgres
-      - POSTGRES_USER=postgres
-    volumes:
-      - postgresdb:/usr/src/data/db
-    ports:
-      - 5432:5432
-  nginx:
-    image: nginx:mainline-alpine3.18-slim
-    ports:
-      - 80:80
-    volumes:
-      - ./nginx/nginx.conf.prod:/etc/nginx/conf.d/nginx.conf
-    depends_on:
-      - postgresql
-      - php8.1
-  adminer:
-    image: adminer:latest
+    php:
+        build: 
+            ./php
```

```suggestion
         сontext: php
```

PhpStrom пишет Scalar value is not allowed here, лучше юзай через context